superset/views/core.py

```diff
@@ -243,6 +244,17 @@ def _deserialize_results_payload(
             return json.loads(payload)  # type: ignore
 
 
+def get_cta_schema_name(
+    database: Database, user: ab_models.User, schema: str, sql: str
+) -> Optional[str]:
+    func = config.get(
+        "SQLLAB_CTA_SCHEMA_NAME_FUNC"
+    )  # type: Optional[Callable[[Database, ab_models.User, str, str], str]]
```

Comment:
Same as above. Also, we prefer using square brackets when reading configs, i.e. `config["SQLLAB_CTA_SCHEMA_NAME_FUNC"]` to avoid having to check for `None` (doesn't really apply in this case, but it's a convention nonetheless).